daffodil-core/src/test/scala/org/apache/daffodil/core/util/TestUtils.scala

```diff
@@ -389,7 +388,7 @@ class Fakes private () {
     override def isError: Boolean = false
 
     override def tunables: DaffodilTunables = DaffodilTunables()
-    override def variableMap: VariableMap = VariableMapFactory.create(Nil)
+    override def variableMap: VariableMap = VariableMap(Nil)
```

Comment:
There was some reason for the factory..... I can't recall. Maybe serialization and hashmap related? In any case if tests pass now, and you can serialize/deserialize processors it's no longer needed.

Reply:
Ah. I was confusing this with VariableBox, which is associated with unparser suspensions in some way.